static/src/stylesheets/module/content/_block-level-sharing.scss

```diff
@@ -238,48 +238,36 @@
 // Gallery
 
 .block-share--gallery {
-    margin-top: $gs-baseline;
-    white-space: nowrap;
-    clear: both;
-
-    @include mq(tablet, desktop) {
-        position: absolute;
-        top: -$gs-baseline / 3;
-        right: 0;
-
-        .gallery-lightbox & {
-            right: $gs-gutter / 2;
-        }
-    }
+    position: static;
 
     .block-share__item {
-        transition: opacity .15s linear;
-        .inline-icon {
-            fill: #ffffff;
-        }
-    }
-
-    .block-share__item--facebook {
-        background-color: $social-facebook;
-
-        &:hover {
-            background-color: colour-hover($social-facebook);
-        }
-    }
-
-    .block-share__item--twitter {
-        background-color: $social-twitter;
+        position: relative;
+        background-color: transparent;
+        border: 1px solid $neutral-2;
+        box-sizing: border-box;
+        transition: border-color .15s ease-out;
+        width: 32px;
+        height: 32px;
 
         &:hover {
-            background-color: colour-hover($social-twitter);
+            background-color: transparent;
+            border-color: $neutral-6;
         }
-    }
-
-    .block-share__item--pinterest {
-        background-color: $social-pinterest;
 
-        &:hover {
-            background-color: colour-hover($social-pinterest);
+        &.inline-icon, .inline-icon {
```

Comment:
Separate these onto different lines

```css
&.inline-icon,
.inline-icon {
```

Although why the need for both? Seems like messy html